internal/dag/builder_test.go

```diff
@@ -1663,29 +1663,6 @@ func TestDAGInsert(t *testing.T) {
 				},
 			),
 		},
-		"insert ingress w/ tls with different secure port": {
```

Comment:
Around line 1375 you'll find
```go
        tests := map[string]struct {
                *Builder
                objs []interface{}
                want []Vertex
        }{
```
now there is no need to provide a custom `*Builder` as part of the fixture -- as there is nothing to customise -- this field can probably be removed and the logic on like 3688 cleaned up.

Reply:
Removed the `*Builder` and just made a new Builder in the actual execution loop down below.